src/com/lftechnology/java/training/alina/collections/generics/wildcards/CustomerUtil.java

```diff
@@ -0,0 +1,57 @@
+
+package com.lftechnology.java.training.alina.collections.generics.wildcards;
+
+import java.util.ArrayList;
+import java.util.List;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.util.List'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)